dtfs-central-api/test-helpers/test-data/tfm-facility.ts

```diff
@@ -2,18 +2,31 @@ import { ObjectId } from 'mongodb';
 import { TfmFacility } from '@ukef/dtfs2-common';
 import { generateMockPortalUserAuditDatabaseRecord } from '@ukef/dtfs2-common/change-stream/test-helpers';
 import { aFacility } from './facility';
+import { KeyingSheetCalculationFacilityValues } from '../../src/types/tfm/tfm-facility';
+
+const facility = aFacility();
+const { dayCountBasis, interestPercentage, coverPercentage, value } = facility;
 
 export const aTfmFacility = (): TfmFacility => {
   const tfmFacilityId = new ObjectId();
 
   return {
     _id: tfmFacilityId,
     facilitySnapshot: {
-      ...aFacility(),
+      ...facility,
       _id: tfmFacilityId,
     },
     amendments: [],
     tfm: {},
     auditRecord: generateMockPortalUserAuditDatabaseRecord(new ObjectId()),
   };
 };
+
+export const keyingSheetCalculationFacilityValues = {
+  coverEndDate: new Date(),
+  coverStartDate: new Date(),
+  dayCountBasis,
+  interestPercentage,
+  coverPercentage,
+  value,
+} as KeyingSheetCalculationFacilityValues;
```

Comment:
We should not be using `as` here, that is dangerous, it says 'whatever this actually is please treat it as this', instead for type safety, you should do the following:
```suggestion
export const keyingSheetCalculationFacilityValues: KeyingSheetCalculationFacilityValues = {
  coverEndDate: new Date(),
  coverStartDate: new Date(),
  dayCountBasis,
  interestPercentage,
  coverPercentage,
  value,
};
```
That way if the object falls out of line with the type you will be made aware because it will cause a type error.

(I would prefer this is a function for consistency and also prefer it was not coupled to other functions which make it hard to see where values actually come from and what value they have, it's just unnecessary and overly complicated in my opinion, but I'll approve as long as you do the typing correctly)